python/doc/examples/numerical_methods/optimization/plot_optimization_pagmo.py

```diff
@@ -0,0 +1,71 @@
+"""
+Multi-objective optimization using Pagmo
+========================================
+"""
+# %%
+# In this example we are going to explore optimization using OpenTURNS' `Pagmo <https://esa.github.io/pagmo2/>`_ interface.
+
+# %%
+from __future__ import print_function
+import openturns as ot
+from openturns.viewer import View
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# List available algorithms
+for algo in ot.Pagmo.GetAlgorithmNames():
+    print(algo)
+
+# %%
+# More details on Pagmo algorithms are available `here <https://esa.github.io/pagmo2/overview.html#list-of-algorithms>`_ .
+
+# %%
+# Create the problem, from ZDT test suite
+f = ot.SymbolicFunction(['x1', 'x2'], ['x1', 'var g := (1.0+9*x2); g * (1.0 - sqrt(x1 / g))'])
+zdt1 = ot.OptimizationProblem(f)
+zdt1.setBounds(ot.Interval([0.0] * 2, [1.0] * 2))
+
+# %%
+# Create the initial population
```

Comment:
Please create a sentence and end it with a point. May you explain what the sample is for in this case?